website/src/corso-announcement-free-backup-for-microsoft-365.html

```diff
@@ -52,13 +52,31 @@ <h3 class="mb-3 text-3xl leading-normal font-medium text-white">A home to call y
                     <div class="lg:col-span-8 md:col-span-6">
                         <div class="p-6 rounded-md shadow dark:shadow-gray-800">
 
-                            <img src="assets/images/blog/office_desk.jpg" class="rounded-md" alt="Office desk with laptop and monitor">
+                            <div id="output" class="content markdown-body"><h1 class="mb-6 mt-8 md:text-4xl text-3xl md:leading-normal leading-normal font-bold" id="a-home-to-call-your-own-the-need-for-your-own-backups-in-it">A Home to Call Your Own: The Need for Your Own Backups in IT</h1>
+                                <p class="text-base pb-4">Have you had it with Google sheets? So have I. Excel is my home. It’s where I write all my best formulae. And what about PowerPoint? The way it just finds stock photos for you? The automatic ‘alternative designs for this slide’ button? It’s too good. I can’t give up Microsoft 365. </p>
+                                <p class="text-base pb-4">If you did some work today, there’s a very good chance you opened a Microsoft tool. M365 is used by <a href="https://www.statista.com/statistics/983321/worldwide-office-365-user-numbers-by-country/">more than a million</a> companies worldwide, and nearly 880,000 companies in the U.S. use the software suite. But with that widespread usage comes risk, business-critical data is at risk of loss or corruption, if not securely backed up and protected. </p>
+                                <h2 class="mb-4 text-3xl leading-normal font-medium pt-6" id="the-problem-with-built-in-backups">The problem with built-in backups</h2>
+                                <p class="text-base pb-4">A couple of years back I took the time to get the AWS ‘baby cert,’ their first certification. The focus of the material surprised me, along with learning about the most popular AWS products and their benefits, you had to learn cold all the things that AWS will not do for you. AWS won’t alert you to poor performance on your applications. It won’t automatically scale down your instances. And while AWS and the other public cloud providers completely meet their promised SLA, they do not promise to deliver the backups that you expect them to deliver. </p>
```

Comment:
I again see what you are saying here but this is not technically accurate given that AWS provides an AWS backup solution. Referring to something like https://www.microsoft.com/en-us/security/blog/2018/06/19/driving-data-security-is-a-shared-responsibility-heres-how-you-can-protect-yourself/ would be more relevant.

Reply:
added/revised